### PR TITLE
VTP + PVD output for particles if output option is set as PVD

### PR DESCRIPTION
**Pull request by Thomas-TK**

### CODE REVIEW

**Comment by Thomas-TK** [comment]
I will have a look at the benchmarks !

**Comment by Thomas-TK** [comment]
The failing benchmarks use PVD as output option and therefore no VTKs are written anymore. PVD and VTPs instead. Benchmarks must be updated.

File: FEM/problem.cpp [hunk 1]
```diff
@@ -753,6 +753,15 @@ inline int Problem::AssignProcessIndex(CRFProcess* m_pcs, bool activefunc)
...
+		COutput* out = OUTGetRWPT("PARTICLES");
+		if (out != NULL)
+		{
+			if (out->dat_type_name.compare("PVD") == 0)
+				DATWriteParticleVTPFile(0);
+			else
+				DATWriteParticleFile(0);
+		}
+		else
 		DATWriteParticleFile(0); // YS
```
Comment (norihiro-w) [inline FEM/problem.cpp]: can you add an indent here?

**Comment by norihiro-w** [comment]
i have only one tiny comment. 👍  if the benchmark is green

**Comment by bilke** [comment]
@Thomas-TK Yes Wenginq is correct, change this line: https://github.com/ufz/ogs5/blob/master/scripts/cmake/benchmarks/YS.cmake#L24

**Comment by Thomas-TK** [comment]
Thank you

**Comment by wenqing** [comment]
@norihiro-w  Now the failed benchmarks are those THM related due to the merging of ufz/ogs5-benchmarks_ref#24. Therefore, I merged this PR.  I will revised  PR #105 soon.

**Pull request merged**